src/agent/src/rpc.rs

```diff
@@ -1846,9 +1876,17 @@ pub async fn start(
     server_address: &str,
     init_mode: bool,
 ) -> Result<TtrpcServer> {
+    #[cfg(feature = "sealed-secret")]
+    let client = {
+        let c = ttrpc::asynchronous::Client::connect(CDH_ADDR)?;
+        Some(CDHClient::new(c))
+    };
```

Comment:
Only a little personal views :)

I found that the `ttrpc::asynchronous::Client::connect(CDH_ADDR)?` and `CDH_ADDR` const here are not needed to be exposed in `rpc.rs`. How about defining a `CDHClient::new()`, and wrap initialization of the ttrpc client code into that function? Then we can make the feature codes and non-feature codes more distinguishable.

Reply:
fixed